dds_cli/__main__.py

```diff
@@ -97,7 +98,10 @@ def dds_main(ctx, verbose, log_file):
         if not any(x in sys.argv for x in ["--config", "-c", "--username", "-u"]):
             config_file = pathlib.Path().home() / pathlib.Path(".dds-cli.json")
             if not config_file.is_file():
-                raise exc.ConfigFileNotFoundError(config_file)
+                LOG.error(
+                    f"Username omitted and config file not specified nor found at default path: {config_file}"
+                )
+                sys.exit(1)
```

Comment:
I thought that `sys.exit(1)` would give a really ugly message but it appears it doesn't. So 👍🏻